### PR TITLE
Fix Xenial ports url

### PR DESCRIPTION
**Pull request by LocutusOfBorg**

### CODE REVIEW

File: distributions/arm64 [hunk 1]
```diff
@@ -54,11 +54,10 @@ extramirrors: deb http://security.debian.org/ wheezy/updates main contrib non-fr
 
 [xenial]
 suite: xenial
-mirror: http://archive.ubuntu.com/ubuntu
+mirror: http://ports.ubuntu.com
 components: main restricted universe multiverse
-extramirrors: deb http://archive.ubuntu.com/ubuntu/ xenial-updates main restricted universe multiverse
-              deb http://security.ubuntu.com/ubuntu xenial-security main restricted universe multiverse
-              deb http://debomatic-amd64.debian.net/debomatic/xenial xenial main
```
Comment (dktrkranz) [inline distributions/arm64:61]: We need this line, after s/amd64/arm64/ it

File: distributions/armhf [hunk 2]
```diff
@@ -54,11 +54,10 @@ extramirrors: deb http://security.debian.org/ wheezy/updates main contrib non-fr
 
 [xenial]
 suite: xenial
-mirror: http://archive.ubuntu.com/ubuntu
+mirror: http://ports.ubuntu.com
 components: main restricted universe multiverse
-extramirrors: deb http://archive.ubuntu.com/ubuntu/ xenial-updates main restricted universe multiverse
-              deb http://security.ubuntu.com/ubuntu xenial-security main restricted universe multiverse
-              deb http://debomatic-amd64.debian.net/debomatic/xenial xenial main
```
Comment (dktrkranz) [inline distributions/armhf:61]: We need this line, after s/amd64/armhf/ it

File: distributions/powerpc [hunk 3]
```diff
@@ -54,11 +54,10 @@ extramirrors: deb http://security.debian.org/ wheezy/updates main contrib non-fr
 
 [xenial]
 suite: xenial
-mirror: http://archive.ubuntu.com/ubuntu
+mirror: http://ports.ubuntu.com
 components: main restricted universe multiverse
-extramirrors: deb http://archive.ubuntu.com/ubuntu/ xenial-updates main restricted universe multiverse
-              deb http://security.ubuntu.com/ubuntu xenial-security main restricted universe multiverse
-              deb http://debomatic-amd64.debian.net/debomatic/xenial xenial main
```
Comment (dktrkranz) [inline distributions/powerpc:61]: We need this line, after s/amd64/powerpc/ it

**Comment by LocutusOfBorg** [comment]
fixed :)

**Pull request merged**